R/oa_fetch.R

```diff
@@ -392,7 +392,10 @@ oa_request <- function(query_url,
 #' For example: "oa_status" for works.
 #' See more at <https://docs.openalex.org/how-to-use-the-api/get-groups-of-entities>.
 #' @param search Character. Search is just another kind of filter, one that all five endpoints support.
-#' But unlike the other filters, search doesn't require an exact match.
+#' But unlike the other filters, search does NOT require an exact match. This is particularly useful to search for authors, because some authors have middle name besides their first and last name. 
+#' Their middle names in a variety of forms may not exist, or may co-exist in OpenAlex datasets (e.g. Phillip H. Kuo, Phillip Hsin Kuo).  
```

Comment:
```suggestion
#' But unlike the other filters, search does NOT require an exact match.
#' This is particularly useful in author queries because many authors have middle names, which may not exist or do so in a variety of forms.  
```
We discuss the example below so I'm trying to avoid repetition here.